test/kubemark/run-e2e-tests.sh

```diff
@@ -33,11 +33,13 @@ export KUBECONFIG="${ABSOLUTE_ROOT}/test/kubemark/resources/kubeconfig.kubemark"
 export E2E_MIN_STARTUP_PODS=0
 
 if [[ -z "$@" ]]; then
-	ARGS='--ginkgo.focus=\[Feature:Performance\]'
+	ARGS='--ginkgo.focus=[Feature:Performance]'
 else
 	ARGS=$@
 fi
 
+ARGS=$(echo $ARGS | sed 's/\[/\\\[/g' | sed 's/\]/\\\]/g')
```

Comment:
AFAIU it treats backslashes we pass as escape characters for brackets and removes them when parsing arguments.